src/components/Plot.js

```diff
@@ -24,6 +26,15 @@ class PlotlyGraph extends React.Component {
   }
 
   render() {
+    if(this.props.data.length === 0) {
```

Comment:
What happens if/when data is not received?
Eg: for given AS we don't have data for one of risk at all (this will probably be a common thing). 
This will spin forever right?